libavcodec/av1dec.c

```diff
@@ -569,6 +569,8 @@ static int get_pixel_format(AVCodecContext *avctx)
 static void av1_frame_unref(AVCodecContext *avctx, AV1Frame *f)
```

Comment:
maybe a better log is "add file grain support to vaapi"
will this impact dxva?

Reply:
This is common part for all av1 hw dec, and the change will happen on some basic struct. So I just want to avoid to mention "vaapi only thing". Anyway, I will think about it, to let the message more acceptable to community.